packages/react-components/src/components/NotificationStack.tsx

```diff
@@ -301,6 +301,14 @@ export interface NotificationStackStrings {
    * Message shown in notification when together mode ends
    */
   togetherModeEnded?: NotificationStrings;
+  /**
+   * Message shown in notification when there was an error starting transcription
```

Comment:
if this is specific to failing to start let's make it that, in case we have transcriptionErrors in future that potentially could happen, e.g. if the transcription service goes down

  transcriptionFailedToStart?: NotificationStrings;

Reply:
I believe the intent here is that Contoso can show this UI whenever something goes wrong with transcription, whether its call automation, or their server this is something to let the people in the call know something broke and it stopped. I wil change the comment